bot/app/database/db_objects.py

```diff
@@ -0,0 +1,212 @@
+#-*- coding: utf-8 -*-
+
+"""Classes representing database items 
+
+To Do :
+
+    playsAt:
+        * insert functionality into Event and Artist Classes
+
+    all:
+        * fix documentation strings in functions to include: 
+            * what database item
+            * which operations incl. parameters
+        * add logging, catch errors to log
+"""
+
+import sqlite3
+import logging
+import os
+
+logging.basicConfig(os.path.abspath('../log.txt', level=logging.DEBUG))
+
+DB_NAME = 'data.sqlite'
+
+class Event:
+
+    """An instance of the Event relation 
+
+    Arguments:
+        * name (str): event's name
+        * date (str): event's date
+        * time (str): event's start time
+        * admission (str): admission required at entry
+        * description (str): short descriptive text. keep it 150 characters or less
+        * location (str): where the event takes place
+        * pic_id (int): ID for pictures repository. 6 characters long, starts with 0.
+    """
+
+    def __init__(self, name, date, time, admission, description, location, pic_id):
+        
+        self.name = name
+        self.date = date
+        self.time = time
+        self.admission = admission
+        self.description = description
+        self.location = location
+        self.pic_id = pic_id
+    
+    def insert_event(self):
+        
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+
+        try:
+            curs.execute("""
+                insert into Events values (?,?,?,?,?,?,?)""",
+                (self.name, self.date, self.time, self.admission, self.description, \
+                self. location, self.pic_id))
+            self.conn.commit()
+        except sqlite3.IntegrityError as e:
+            logging.error(e)
+        
+        curs.close()
+        conn.close()
+
+    def is_event(self):
+
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+
+        search = curs.execute("""select from Artists where aName=?""", (self.name))
+        
+        curs.close()
+        conn.close()
+
+        return search is not None
```

Comment:
Don't you need to `return search.fetchone() is not None`?

Reply:
fixed